.github/workflows/docker-release.yml

```
@@ -12,8 +12,20 @@ on:
       - 'yarn.lock'
 
 jobs:
+  check-env:
+    runs-on: ubuntu-latest
+    outputs:
+      check-docker: ${{ steps.check-docker.outputs.defined }}
+    steps:
+      - id: check-docker
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        if: "${{ env.DOCKER_USERNAME != '' }}"
```

```suggestion
        if: ${{ env.DOCKER_USERNAME != '' }}
```
I think quotes can be skipped